packages/react-table/src/components/Table/utils/headerUtils.tsx

```diff
@@ -109,7 +109,7 @@ const mapHeader = (column: ICell, extra: any, key: number, ...props: any) => {
     property:
       (typeof title === 'string' &&
         title
-          .toLowerCase()
+          .toUpperCase()
           .trim()
           .replace(/\s/g, '-')) ||
```

Comment:
I wonder if we could employ a hash function?
```js
/**
 * Calculate a 32 bit FNV-1a hash
 * Found here: https://gist.github.com/vaiorabbit/5657561
 * Ref.: http://isthe.com/chongo/tech/comp/fnv/
 *
 * @param {string} str the input value
 * @param {boolean} [asString=false] set to true to return the hash value as 
 *     8-digit hex string instead of an integer
 * @param {integer} [seed] optionally pass the hash of the previous chunk
 * @returns {integer | string}
 */
function hashFnv32a(str, asString, seed) {
    /*jshint bitwise:false */
    var i, l,
        hval = (seed === undefined) ? 0x811c9dc5 : seed;

    for (i = 0, l = str.length; i < l; i++) {
        hval ^= str.charCodeAt(i);
        hval += (hval << 1) + (hval << 4) + (hval << 7) + (hval << 8) + (hval << 24);
    }
    if( asString ){
        // Convert to 8 digit hex string
        return ("0000000" + (hval >>> 0).toString(16)).substr(-8);
    }
    return hval >>> 0;
}
```

Reply:
Tbh, I feel like that's unnecessary obfuscation (from a debugging standpoint the rowData object becomes unreadable), the problem is with `id` and `cells` since those are single words and thus not in camelCase like the rest. If we make sure the column names are never lowercase/camelCase then we're fine.

Reply:
If you'd like it to be debuggable just append the hash to column name. Unfortunately, we can't "make sure the column names are never lowercase/camelCase" and it's likely that there are many instances of this out in the wild.

Reply:
This is happening because of variables we're inserting into the rowData object, we have full control over that, don't we?

Reply:
I don't know. @jschuler do you know?